packages/scaffolding-cli/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@amidostacks/scaffolding-cli",
-  "version": "1.22.0",
+  "version": "1.23.0",
```

Comment:
Did you run `npm run version`? Or was this a manual bump? We should have the changelog generated and the package-lock.json files should also be bumped when lerna is run.